Ginger/Ginger/AnalyzerLib/AnalyzerPage.xaml.cs

```diff
@@ -143,17 +144,19 @@ public async Task AnalyzeWithoutUI()
         }
 
         private async Task Analyze()
-        {         
-                // Each analyzer will set to true once completed, this is prep for multi run in threads for speed
-                BusyInProcess = true;
-                mAnalyzerCompleted = false;
-                mAnalyzeDoneOnce = true;
-                try
+        {
+            // Each analyzer will set to true once completed, this is prep for multi run in threads for speed
+            BusyInProcess = true;
+            mAnalyzerCompleted = false;
+            mAnalyzeDoneOnce = true;
+            try
+            {
+                if (mAnalyzeWithUI)
+                {
+                    SetStatus("Analyzing Started");
+                }
```

Comment:
why change need to be on the AnalyzerPage? what if we do not show UI, we still using the Page for running Analyzer?